pypit/scripts/setup.py

```diff
@@ -40,53 +42,83 @@ def parser(options=None):
 
 def main(args):
 
+    import pdb
```

Comment:
Can this be removed? Or can we use `import pdb as debugger` (or our other try except clause including xdb)?